Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/TMP_KeyboardInputField.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    [RequireComponent(typeof(TMP_InputField))]
+    public class TMP_KeyboardInputField : KeyboardInputFieldBase<TMP_InputField>
```

Comment:
Not sure what the naming convention is in MRTK for underscore class names but may be an issue

Reply:
The original TMP_InputField also has an underscore, why would this be a problem?